package.json

```diff
@@ -335,7 +335,7 @@
     "whatwg-fetch": "^2.0.3"
   },
   "resolutions": {
-    "**/lodash": "4.17.15",
+    "**/lodash": "4.17.19",
```

Comment:
I'm honestly not sure why it's needed _now_, but `yarn install --frozen-lockfile` started failing without this resolution to the latest Lodash. Also, it seems more correct to resolve it to the current version that we're installing. :shrug:

Reply:
Turns out it was due to the [Lodash dependency upgrade](https://github.com/department-of-veterans-affairs/generator-vets-website/pull/50) on `generator-vets-website`. There hasn't been a release until now that included that change.